src/sdk/pynni/nni/evolution_tuner/evolution_tuner.py

```diff
@@ -299,3 +300,13 @@ def receive_trial_result(self, parameter_id, parameters, value):
 
         indiv = Individual(config=params, result=reward)
         self.population.append(indiv)
+
+    def feed_tuning_data(self, data):
```

Comment:
@xuehui1991 Is this method implemented right?

Reply:
seems not ok.  shall we consider the optimize mode?

Reply:
Seems we cannot feed the data like that. please do not change the "evolution tuner". by the way, what's the meaning of "tuning_data"???

Reply:
Feeding data to evolution_tuner is currently not supported.